src/develop/blends/blendif_lab.c

```diff
@@ -122,7 +122,7 @@ static inline void _blendif_lab_a(const float *const restrict pixels, float *con
 {
   for(size_t x = 0, j = 0; x < stride; x++, j += DT_BLENDIF_LAB_CH)
   {
-    mask[x] *= _blendif_compute_factor(pixels[j + 1] / 128.0f, invert_mask, parameters);
+    mask[x] *= _blendif_compute_factor(pixels[j + 1] / 256.0f, invert_mask, parameters);
```

Comment:
shouldn't that be 255 ?

Reply:
Not sure, I guess it depends on the conventions. Looking at the code of darktable 3.2.1 we have:
```
      scaled[DEVELOP_BLENDIF_A_in]
          =clamp_range_f((input[1]+128.0f)/256.0f, 0.0f, 1.0f); // a scaled to 0..1
      scaled[DEVELOP_BLENDIF_B_in]
          =clamp_range_f((input[2]+128.0f)/256.0f, 0.0f, 1.0f);                 // b scaled to 0..1
```
It looks like darktable is using the convention of a and b values between -128.0f and 128.0f.